src/package/components/LessonOverviews/components/PaginatedLessonOverviews/index.js

```diff
@@ -44,13 +44,13 @@ const PaginatedLessonOverviews = ({
         <Maybe condition={hasMoreThanOnePage}>
           <div className='pa3'>
             <ReactPaginate
```

Comment:
It would probably be good to abstract this out into an egghead-ui `Pagination` component at some point instead of using the lib directly